README.md

```diff
@@ -75,6 +76,18 @@ The arguments are the following:
         Max number of fixes per rule. Default: Integer.MAX_VALUE (or all)
         (default: 2147483647)
 
+  [--maxFilesPerSegment <maxFilesPerSegment>]
+        Max number of files per loaded segment for segmented repair. It should
+        be >= 3000 files per segment. Default: 6500 (256mb Jvm) .  (default:
+        6500)
+
+  [--repairStrategy <repairStrategy>]
+        Type of repair strategy. DEFAULT - load everything without splitting up
+        the folder in segments, SEGMENT - splitting the folder into smaller
+        segments and repair one segment at a time (need to specify
+        --maxFilesPerSegment if not default). Default: DEFAULT (default:
+        DEFAULT)
```

Comment:
I have three points to mention here (that should be fixed first in the code, and then here, with the output from the code):

- You shouldn't write the default values because they are already shown automatically.

- The arg `repairStrategy` should appear before `maxFilesPerSegment` since first one decides the strategy for the repair, and if the segmented one is chosen, one sets the max files per segment.

- Why should the max files per segment be >= 3000?

Reply:
Fixed :).